test/support/cases/mix_case.ex

```diff
@@ -0,0 +1,59 @@
+defmodule Torch.MixCase do
+  @moduledoc false
+
+  use ExUnit.CaseTemplate
+
+  using do
+    quote do
+      import Torch.MixCase
+    end
+  end
+
+  @doc false
+  # Generates tests on the given mix task to ensure it handles errors properly
+  defmacro test_mix_config_errors(task) do
+    quote location: :keep do
```

Comment:
Really interesting, I didn't know about maintaining the stack-trace with `location: :keep` 👍